backend/logic/users/users.js

```diff
@@ -61,22 +65,31 @@ exports.insertStudentUser = (username, password, email, id) => {
 				if (error) { res(logError(connection, error)); return; }
 				// Test and insert into users table
 				connection.query('INSERT INTO users VALUES(?, ?, ?);',
-				                 [username, email, hash],
-				                 function (error, results, fields) {
-					if (error) { res(logError(connection, error)); return; }
-					// Test and insert into students table
-					connection.query('INSERT INTO students VALUES(?, ?);',
-					                 [id, username],
-					                 function (error, results, fields) {
-						if (error) { res(logError(connection, error)); return; }
-						// Commit transaction
-						connection.commit(function(error) {
-							if (error) { res(logError(connection, error)); return; }
-							connection.release();
-							res('true');
-						});
+					[username, email, hash],
+					function (error, results, fields) {
+						if (error) {
+							res(logError(connection, error));
```

Comment:
Again good to see you're looking to handle errors, it makes your code more solid. Just noticed that you used resolve instead of reject, was this intentional?

Reply:
This is one of the things about error handling to discuss during the meeting.  Steven suggested resolving with a status message. Other options also being considered. Will be brought up during the meeting